config/initializers/surveys.rb

```diff
@@ -108,8 +107,12 @@ def set_tags
     scope :course_data_questions, ->{where("course_data_type <> ''")}
     serialize :alert_conditions, Hash
 
-    def self.for_conditionals
-      where("conditionals IS NULL OR conditionals = ''")
+    def self.for_conditionals(question_id)
+      unless question_id.nil?
+        where.not(id: question_id).where("conditionals IS NULL OR conditionals = ''")
```

Comment:
Would it work to just use `where.not(id: question_id)` for every query? If question_id is `nil`, it should have the same behavior, I think.

Reply:
I meant, I think it would work to use the entire thing, `where.not(id: question_id).where("conditionals IS NULL OR conditionals = ''")`, for every query. That ought to work even when `question_id` is nil.

Reply:
Got it, yes it will work :)